packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2103,12 +2105,22 @@ export const audiusBackend = ({
     const timestamp = notification.actions[0].timestamp
     return {
       groupId: notification.group_id,
-      timestamp: new Date(timestamp * 1000).toString(),
+      timestamp,
       isViewed: notification.seen_at == null,
       id: `timestamp:${timestamp}:group_id:${notification.group_id}`
     }
   }
 
+  function mapIdentityNotification(
+    notification: IdentityNotification
+  ): Notification {
+    const { timestamp, ...restNotification } = notification
+    return {
+      ...restNotification,
+      timestamp: Date.parse(timestamp) / 1000
```

Comment:
may be worth leaving a comment here explaining this conversion??

Reply:
totally!